src/tutorials/0004/07.vue

```diff
@@ -139,10 +142,6 @@ const validate = async (result, ipfs) => {
       logDesc: "Here's what was returned by `ls` in your root directory. Notice how directories have a type of `1` while files have a type of `0`.",
       log: logRoot
     }
-  } else if (result.error) {
-    return { error: result.error }
-  } else {
-    return { fail: 'Something we haven\'t anticipated is wrong. :(' }
```

Comment:
@dominguesgm Can you please confirm which error message now built into Lesson.vue will get surfaced now that this is gone? (This question applies across multiple lessons.) We have a number of different default unknown error messages across various lessons and I want to confirm we're putting the most appropriate one in the right place in the new shared code. Thanks!

Reply:
In this specific lesson, with the logic in the previous 2 if blocks, this final else would never be reached. However, if there is missing validation, and the whole validation function is executed without a `return` statement being hit, we get the following message:

```javascript
} else if (test === undefined) {
  let validationErrorMessage = `You may have uncovered a bug in our validation code. Please help us improve this lesson by [opening an issue](${this.validationIssueUrl}) noting that you encountered a validation error and pasting in the code you submitted.`
  // Our validation did not return anything and the original result is also not an error.
  // This may be the result of a missing validation case + not returning anything by default
  test = {
    fail: validationErrorMessage
  }
}
```

This is our new equivalent to _'Something we haven't anticipated is wrong. :('_